templates/taxi/manufacturer_delete.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Manufacturer</h1>
+  <p>
+    Are u sure, that u want to delete manufacturer {{ manufacturer }}?
+  </p>
+  <p><i>All info about this manufacturer, and cars of this manufacturer will be irretrievably deleted</i></p>
+  <form action="" method="post">  
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-secondary">
+    <a href="{% url 'taxi:manufacturer-list' %}" class="btn btn-danger">Cancel</a>
```

Comment:
The 'Cancel' button should lead to the previous page the user was on, not a fixed URL like 'manufacturer-list'. Consider using JavaScript to go back to the previous page or a more dynamic approach.